bessctl/cli.py

```diff
@@ -231,11 +238,16 @@ def list_matched(self, line, filters):
 
         ret = []
         ret_low = []
-        for cmd, score in matched_list:
-            if score == max_score:
-                ret.append(cmd)
-            else:
-                ret_low.append(cmd)
+
+        ret = [m[0] for m in matched_list if m[1] == max_score]
+        ret_low = [m[0] for m in matched_list if m[1] != max_score]
+
+        # Find exact matches without ambiguity
+        if filter == 'full' and len(ret) > 1:
+            full_matches = [m for m in matched_list if m[1] == max_score]
+            full_matches.sort(key=itemgetter(2), reverse=True) # sorted by exact score
```

Comment:
Maybe it's just me, but `key=itemgetter(2)` (which requires external module) sounds less familiar to me than `key=lambda m: m[2]`

Reply:
Using itemgetter (and attrgettr) in sort is kind of a standard idiom (see https://docs.python.org/2/howto/sorting.html). I'm not sure if the sort code optimizes these specially. If it does that might be a reason to use itemgetter, if not they're kind of six-of-one half-a-dozen-of-the-other.